heartofthealien/heartofthealien-1.2.2.recipe

```diff
@@ -4,9 +4,9 @@ HOMEPAGE="http://hota.sourceforge.net/"
 COPYRIGHT="2004 Gil Megidish"
 LICENSE="GNU GPL v2"
 REVISION="1"
-srcGitRev = "8f1f525fcf50c4610302e870c85625c8ea2f33eb"
+srcGitRev="8f1f525fcf50c4610302e870c85625c8ea2f33eb"
 SOURCE_URI="https://github.com/ytmytm/beos-heartofthealien/archive/$ srcGitRev.tar.gz"
-SOURCE_FILENAME = "heartofthealien- $ PORTVERSION ~ $ srcGitRev.tar.gz"
+SOURCE_FILENAME="heartofthealien- $ PORTVERSION ~ $ srcGitRev.tar.gz"
```

Comment:
No spaces in SOURCE_FILENAME too. It should be:
```
SOURCE_FILENAME="heartofthealien-$portVersion~$srcGitRev.tar.gz"
```
exacly the way I wrote in my first review.

Reply:
Sorry. Edited

Reply:
Please have a look at a few other recipes. I've learned a lot just by reading existing recipes.